### DISCUSSION TITLE
🚀 Celebrating open-source RudderStack v1.0, thanks to 170+ contributors

### DISCUSSION THREAD

**Discussion by gitcommitshow** [rudderlabs/rudder-server#2401] [Changelogs]
:rocket: Happy to announce open-source [**RudderStack v1.0**](https://github.com/rudderlabs/rudder-server/releases), a stable and production-ready Customer Data Platform for developers

Join me in thanking 170+ contributors who made RudderStack the leading open-source CDP for developers over the past 3 years

### What can you do with open-source RudderStack?
For those new community members wondering what they can do with open-source RudderStack
You can create event streams that collect data from multiple web/mobile apps and backends with ordering precision and high availability.
If you are thinking of converting user events into actionable insights, RudderStack is your first step towards building that reliable modern data stack. By self-hosting RudderStack, you get full control over your data.
Check out the [**complete feature list and supported sources/destinations here**](https://github.com/rudderlabs/rudder-server/discussions/2392) for open-source RudderStack.

### If you are an existing user
You do not need to worry about the compatibility while you upgrade. There are no breaking changes in migrating from v0.x to v1.x. The RudderStack project had achieved stability a long time back and 18000+ sites/apps use it in production already. We are marking the latest version v1.0 now as per SemVer specifications i.e. **we want to share our commitment towards stability and backward compatibility for major versions of the project**

### Thanks to contributors
:pray: We would like to thank all the 170+ contributors who contributed to various RudderStack open-source repositories making RudderStack the leading open-source CDP for developers. Let's take a moment to appreciate all the contributors listed below 👇 

@ameypv-rudder,@arnab-p,@psrikanth88,@desusai7,@ruchiramoitra,@saikumarrs,@arnabp92,@sayan-rudder,@sayan-mitra,@achettyiitr,@SumanthPuram,@Veenap,@utsabc,@1abhishekpandey,@chandumlg,@arajguha,@prabrisha-rudder,@shrouti1507,@lokey,@soumyadebm,@sivashanmukh,@abandonned,@pallabmaiti,@dhawal1248,@Jayachand,@ishachopra0401,@akashrpo,@dipanjanb,@cisse21,@prabrishac,@Sampath-vi,@lvrach,@dhawal-rudder,@gane5hvarma,@fxenik,@snarkychef,@DipanjanAtRudderLabs,@ItsSudip,@bipin-rudder,@esadek,@cwray-tech,@MoumitaM,@saivarunr,@Shivam4415,@atzoum,@pfd,@dalyd14,@Sidddddarth,@vyeshwanth,@yashasvibajpai,@sajalrudderlabs,@thtmnisamnstr,@Thrinadh-Kumpatla,@fracasula,@kuldeep0020,@saurav-malani,@sanpj2292,@rohithkaza,@ryanmccrary,@neeravbm,@sauravlal-rudderstack,@yields,@bwalvoord,@Gauravudia,@itsdebs,@rahulkumark,@ashishRudder,@rsathishkumar,@swaChan12,@ujjwal-ab,@BonapartePC,@mcchran,@pierreavizou,@calvinfo,@Rowno,@pChondros,@koladilip,@mihir-4116,@rajvarkala,@abhimanyubabbar,@gpxl,@AkhileshK8y,@AndrewBarba,@ianstormtaylor,@nishantsharma,@rspriyamdixit,@Manashi-1881,@gmathieu,@pooyaj,@rajeshneti,@chrikar,@hugecannon,@Amol-Thakur,@sajalmohanta,@benedikt,@Gauravp15,@DileepPatchigolla,@jdreamerz,@ssbeefeater,@sandeepdsvs,@jgrau,@shashank1992,@benjaminhoffman,@donald-stolz,@kilajyot,@ameyvarangaonkar,@Abhinav796,@mparaz,@keithbrink,@mgz,@parth-mahajan,@f2prateek,@alan-cooney,@ariel-talent-fabric,@chandlervdw,@gassiss,@yevhenvolchenko,@jlee9595,@akshayagarwal,@avinder-red-crackle,@akagr,@Juneezee,@masonhensley,@raman2805,@sourya-deepsource,@gitcommitshow,@RohitKhatta,@lxcid,@daniel-statsig,@bonsaiben,@ayush-mehra,@ther0y,@masad-frost,@lennypisano,@aultimus,@lookering,@Savialobo,@dpatchigolla,@coreyhulen,@deepakrai9185720,@WFrancois,@henricook,@Kannaj,@kevburnsjr,@pandres95,@pbnj,@satishrudderstack,@thoqbk,@cpard,@diptag,@shvetzen,@aashishmalik,@anantjain45823,@atomicman57,@orrc,@gkcgautam,@yeouchien,@migonin,@GSokol,@shreya-yadu,@safaiyeh,@rajeshkp,@code-constructor,@bdashrad,@shekhar-rudder,@vizeke,@BryanJBryce,@dominicbarnes,@ctison,@erinkcochran87,@harshavardhana,@RanjeetMishra,@etler,@Zhong-z,@alex-lzl,@savy-91,@yakkomajuri,@tomasol

**Comment by cwray-tech** [comment]
Congrats @gitcommitshow ! Happy to have been a tiny part of your success. Good luck as you continue!

**Reply by gitcommitshow** [reply]
Thank you for your contribution 🙏

**Comment by gitcommitshow** [comment]
Here’s a visualization of how RudderStack code has progressed thanks to all the contributors.
Turn the sound ON, volume UP and let's get the party started  🎉

https://user-images.githubusercontent.com/56937085/189170715-b9c34090-2570-475f-b58a-7317299edbc4.mp4
